datamodel_code_generator/parser/jsonschema.py

```diff
@@ -633,10 +632,21 @@ def parse_raw_obj(self, name: str, raw: Dict[str, Any], path: List[str]) -> None
         self.parse_ref(obj, path)
 
     def parse_raw(self) -> None:
-        raw_obj: Dict[Any, Any] = json.loads(self.text)  # type: ignore
```

Comment:
Always use yaml parser so we don't care about file format.

Reply:
Great :+1: